react/components/EXPERIMENTAL_Table/README.md

```diff
@@ -23,123 +24,300 @@ Example with simple structure:
 
 ##### title
 
-- Control the title which appears on table Header.
-- It receives only strings.
-- If you want to customize it with a component, you can use the `headerRender` prop.
+- Controls the title which appears on the table Header.
+- If you want to customize it with a component, you can use the `headerRenderer` prop.
 
-##### headerRender
+##### width
+
+- Defines a fixed width for the specific column.
+- By default, the column's width is defined to fit the available space without breaking the content.
+
+##### headerRenderer
 
 - Customize the render method of a single header column cell.
 - It receives a function that returns a node (react component).
-- The function has the following params: ({ headerData })
-- Default is render the value as a string.
-- If you have a custom cell component that has a click interaction and at the same time you use the onRowClick Table prop, you might stumble uppon the problem of both click actions being fired. We can work around that by doing a wrapper around cellRenderer to stop click event propagation.
+- The function has the following params: ({ columnData })
+- The default is rendering the value as a string.
 
-##### cellRender
+##### cellRenderer
 
 - Customize the render method of a single column cell.
 - It receives a function that returns a node (react component).
-- The function has the following params: ({ cellData, rowData })
-- Default is render the value as a string.
-- If you have a custom cell component that has a click interaction and at the same time you use the onRowClick Table prop, you might stumble uppon the problem of both click actions being fired. We can work around that by doing a wrapper around cellRenderer to stop click event propagation.
+- The function has the following params: ({ cellData, rowData, rowHeight, selectedDensity })
+  - cellData: the value of the current cell.
+  - rowData: the value of the current row.
+  - rowHeight: current height of the row.
+  - selectedDensity: current table density.
+- The default is rendering the value as a string.
 
-##### hidden
+To illustrate this info, let's suppose we have a list of heroes, each one with properties `name`, `email`, `age` and `country`:
```

Comment:
Note for the future, apply e-commerce related examples.

Reply:
Sure